io.github.piotrek_k._2dTaskBoard.yml

```diff
@@ -0,0 +1,61 @@
+app-id: io.github.piotrek_k._2dTaskBoard
```

Comment:
The id should be `io.github.piotrek_k.2dTaskBoard`

> The domain portion must be in lowercase and must convert dash - to underscore _ and also prefix any intial digits with an underscore _.

Reply:
It was like that before, but I've ran a validator and it said:

> E: io.github.piotrek_k.2dTaskBoard:3: cid-has-number-prefix
>      io.github.piotrek_k.2dTaskBoard: 2dTaskBoard → _2dTaskBoard
>    The component ID contains a segment starting with a number. Starting a segment of the
>    reverse-DNS ID with a number is strongly discouraged, to keep interoperability with other tools
>    such as D-Bus. Ideally, prefix these segments with an underscore.

I used this command:
```
flatpak run --command=flatpak-builder-lint org.flatpak.Builder appstream io.github.piotrek_k._2dTaskBoard.metainfo.xml
```

Reply:
looks like there a contradiction somewhere....

Reply:
Keep it as is, I gave you an exception for appid-url-not-reachable https://github.com/flathub-infra/flatpak-builder-lint/commit/31a3c4537b721c1b3e8220f2e1ec030e9fd10f2b

Reply:
thanks